vault/token_store.go

```diff
@@ -1617,9 +1599,23 @@ func (ts *TokenStore) handleUpdateRevokeAccessor(ctx context.Context, req *logic
 		return nil, err
 	}
 
-	// Revoke the token and its children
-	if err := ts.RevokeTree(ctx, aEntry.TokenID); err != nil {
-		return logical.ErrorResponse(err.Error()), logical.ErrInvalidRequest
+	te, err := ts.Lookup(ctx, aEntry.TokenID)
```

Comment:
`Lookup`, `CreateOr..` and `Revoke`. This pattern is repeated a couple of times. Can we abstract it out?